packages/strapi-design-system/src/Combobox/__tests__/utils.spec.js

```diff
@@ -6,9 +6,48 @@ describe('Combobox | utils', () => {
       const options = [{ props: { children: 'Hamburger' } }, { props: { children: 'Bagel' } }];
       const actual = filterOptions(options, 'ham');
       const expected = [{ props: { children: 'Hamburger' } }];
+
       expect(actual).toEqual(expected);
     });
 
+    describe('should filter the options correctly, with partial term', () => {
+      const options = [
+        { props: { children: 'hicham/project-super-cms' } },
+        { props: { children: 'pierre/project-awesome-cms' } },
+        { props: { children: 'udi/cool-cms' } },
```

Comment:
Not sure that we should keep the Strapier names in the codebase 😅. Can you rename these values please 👍🏽

Reply:
Sorry about that.
Fixed.
Thank you.